Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.cxx

```diff
@@ -281,6 +282,39 @@ void ApplyImageSeriesReaderWorkaround(vtkMRMLVolumeArchetypeStorageNode * storag
     }
   }
 }
+
+//----------------------------------------------------------------------------
+bool IsVolumeCoordinateSystemLeftHanded(const vtkSmartPointer<vtkMatrix4x4>& ijkToRASMatrix)
+{
+  // Check if the determinant of the direction matrix is less than 0.
+  vtkNew<vtkMatrix3x3> direction;
+  for (int i = 0; i < 3; i++)
+  {
+    for (int j = 0; j < 3; j++)
+    {
+      direction->SetElement(i, j, ijkToRASMatrix->GetElement(i, j));
+    }
+  }
+  return direction->Determinant() < 0.;
+}
+
+//----------------------------------------------------------------------------
+void InvertSliceOrderAndFlipKAxisDirection(const vtkSmartPointer<vtkImageData>& imageData,
```

Comment:
Method name containing two actions indicate the something is wrong. In this case, the issue is really small: the method name goes too low level (describes an internal implementation detail that we don't want to commit to). The name proposed here is a bit long, so maybe you can simplify it, but it is at the right abstraction level. If you want you can describe implementation details in the module documentation. 
```suggestion
void FlipIJKCoordinateSystemHandedness(const vtkSmartPointer<vtkImageData>& imageData,
```

Reply:
Thanks for the feedback. I renamed the method as per your suggestions.